tex/chapters/gausss-law/infinite-plane.tex

```diff
@@ -0,0 +1,31 @@
+\section{Infinite plane}
+An infinite plane with charge density per unit of area $\sigma$ in empty space, located on the $XY$-plane is given. Due to the symmetry of a
+system, the electric field is constant and in the direction of the $z$-axis.
```

Comment:
The electric field is not constant. Its strength is constant but its direction is pointing away from the plane.

Reply:
done